tests/presets.spec.js

```diff
@@ -740,6 +740,289 @@ describe('Presets', function() {
+            '[package]',
+            'name = "foo"',
+            'version = "1.0.0"',
+            '',
+            '[dependencies.bar]',
+            'version = "2.0.0"',
+            ''
+          ].join('\n'));
+
+          fs.writeFileSync(this.cargoLock, [
+            '[[package]]',
+            'name = "bar"',
+            'version = "2.0.0"',
+            'source = "registry+https://github.com/rust-lang/crates.io-index"',
+            '',
+            '[[package]]',
+            'name = "foo"',
+            'version = "1.0.0"',
+            'dependencies = [',
+            ' "bar 2.0.0 (registry+https://github.com/rust-lang/crates.io-index)",',
+            ']',
+            '',
+            '[metadata]',
+            '"checksum bar 2.0.0 (registry+https://...)" = "..."',
+            ''
+          ].join('\n'));
+        });
+
+        afterEach(function() {
+          fs.unlinkSync(this.cargoToml);
+          fs.unlinkSync(this.cargoLock);
+          this.cwd.removeCallback();
+        });
+
+        it('should be able to update the version', function(done) {
+          presets.updateVersion.cargo({}, this.cwd.name, '1.0.1', (error) => {
+            m.chai.expect(error).to.not.exist;
+
+            m.chai.expect(fs.readFileSync(this.cargoToml, 'utf8')).to.equal([
+              '[package]',
+              'name = "foo"',
+              'version = "1.0.1"',
+              '',
+              '[dependencies.bar]',
+              'version = "2.0.0"',
+              ''
+            ].join('\n'));
+
+            m.chai.expect(fs.readFileSync(this.cargoLock, 'utf8')).to.equal([
+              '[[package]]',
+              'name = "bar"',
+              'version = "2.0.0"',
+              'source = "registry+https://github.com/rust-lang/crates.io-index"',
+              '',
+              '[[package]]',
+              'name = "foo"',
+              'version = "1.0.1"',
+              'dependencies = [',
+              ' "bar 2.0.0 (registry+https://github.com/rust-lang/crates.io-index)",',
+              ']',
+              '',
+              '[metadata]',
+              '"checksum bar 2.0.0 (registry+https://...)" = "..."',
+              ''
+            ].join('\n'));
+
+            done();
+          });
+        });
+
+      });
+
+      describe('single-quotes in Cargo.toml', function() {
+
+        beforeEach(function() {
+          this.cwd = tmp.dirSync();
+          this.cargoToml = path.join(this.cwd.name, 'Cargo.toml');
+
+          fs.writeFileSync(this.cargoToml, [
+            '[package]',
+            'name = \'foo\'',
+            'version = \'1.0.0\'',
+            '',
+            '[dependencies.bar]',
+            'version = \'2.0.0\'',
+            ''
+          ].join('\n'));
+        });
+
+        afterEach(function() {
+          fs.unlinkSync(this.cargoToml);
+          this.cwd.removeCallback();
+        });
+
+        it('should be able to update the version', function(done) {
```

Comment:
Perhaps this description could be expanded to `it('should be able to update the version (and preserve the single-quotes)', ` ?

Reply:
+1